generator/src/main/java/com/graphicsfuzz/generator/knownvaluegeneration/ExpressionGenerator.java

```diff
@@ -163,6 +168,45 @@ public Expr generateExpr(FactManager factManager,
     return result;
   }
 
+  private Expr generateUniformValue(Value value) {
+    if (value.valueIsUnknown()) {
```

Comment:
Unknown is OK: we can choose a random value for the uniform to be initialized to.